### DISCUSSION TITLE
0.0.1

### DISCUSSION THREAD

**Discussion by MarJose123** [RecruitLab/Recruit#116] [Announcements]
## What's Changed
* Authentication API Route by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/4
* Created a Seeder File and name it as UserSeeder by @YANCZY in https://github.com/OSSAdmiral/Recruit/pull/5
* added Github workflow :zap: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/7
* License by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/8
* Update update-changelog.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/9
* Enhance response by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/10
* Update Seeder Role and Permission by @YANCZY in https://github.com/OSSAdmiral/Recruit/pull/11
* Updated README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/12
* remove sanctum middleware :hammer: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/13
* FilamentPhp Package by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/14
* chore: added pint by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/15
* Module job opening by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/17
* WIP :hammer: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/21
* Create CODE_OF_CONDUCT.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/24
* Create SECURITY.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/25
* Update issue templates by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/26
* Candidates Module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/22
* chore(Change route to default) by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/27
* enabled default password reset by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/28
* fixed code style :hammer: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/29
* added GitHub action for test and fix code style :shirt: :zap: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/31
* applied fix to the test yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/32
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/35
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/37
* Create PULL_REQUEST_TEMPLATE.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/38
* Update PULL_REQUEST_TEMPLATE.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/39
* Create FUNDING.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/44
* 42 testcase login page by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/45
* 23 job candidates module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/33
* Attachment Related list by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/49
* Attachment Related list Enhancement by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/50
* Update FUNDING.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/52
* 19 referrals module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/51
* fix(Model Relation): by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/59
* fix(model attribute): by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/61
* added new column published to career site by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/62
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/63
* 64 db seeder for job opening department module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/65
* Update PULL_REQUEST_TEMPLATE.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/66
* Update bug_report.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/67
* Delete .github/ISSUE_TEMPLATE directory by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/68
* Revert "Delete .github/ISSUE_TEMPLATE directory" by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/69
* Update and rename bug_report.md to bug_report.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/70
* 36 career page by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/58
* #53 auto generate job opening number by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/71
* added candidate auto number and factory by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/72
* Update fix-php-code-style-issues.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/74
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/75
* Update FUNDING.yml by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/76
* Profile page by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/73
* Fix styling :shirt: by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/77
* Settings Selection Page  by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/78
* User resource by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/79
* added user themes by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/80
* Roles and Permissions by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/81
* Bump postcss from 8.4.29 to 8.4.31 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/83
* Bump stefanzweifel/git-auto-commit-action from 4 to 5 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/84
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/85
* Candidate Portal Module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/82
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/89
* Added Change User Name and update password functionality for the the Candidate Portal. by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/90
* added render hook to add the candidate login route in the login page by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/91
* added badge count on modules by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/92
* added favicon by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/93
* Added applied job module by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/94
* Update Career Navigation Logo size by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/95
* Candidate portal invitation by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/96
* Feature add system user via invitation only by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/105
* change the salutation by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/107
* Bump axios from 1.5.0 to 1.6.0 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/108
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/110
* Update README.md by @MarJose123 in https://github.com/OSSAdmiral/Recruit/pull/111
* Bump vite from 4.4.9 to 4.4.12 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/112
* Bump follow-redirects from 1.15.2 to 1.15.4 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/114
* Bump aglipanci/laravel-pint-action from 2.3.0 to 2.3.1 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/113
* Bump vite from 4.4.12 to 4.5.2 by @dependabot in https://github.com/OSSAdmiral/Recruit/pull/115

## New Contributors
* @MarJose123 made their first contribution in https://github.com/OSSAdmiral/Recruit/pull/4
* @YANCZY made their first contribution in https://github.com/OSSAdmiral/Recruit/pull/5
* @dependabot made their first contribution in https://github.com/OSSAdmiral/Recruit/pull/63

**Full Changelog**: https://github.com/OSSAdmiral/Recruit/commits/0.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/OSSAdmiral/Recruit/releases/tag/0.0.1'>0.0.1</a>.</em>

**Comment by Nakanyala** [comment]
Hi, Are these projects still alive? Are we expecting future updates and new versions?

**Reply by MarJose123** [reply]
Hi, Thank you for your interest of this project, unfortunately this project is 50/50. Due time constraint and being a slave in corporate world, I can't fully focus this projects.


You may fork this project or submit PR that will fix or enhancement. I'm happy to review and approve them.
